### PR TITLE
support klusterlet-priority-class

### PR DESCRIPTION
**Pull request by elgnay**
The related issue: https://issues.redhat.com/browse/ACM-6265

### CODE REVIEW

File: pkg/helpers/helpers.go [hunk 1]
```diff
@@ -62,8 +64,9 @@ import (
 const maxConcurrentReconcilesEnvVarName = "MAX_CONCURRENT_RECONCILES"
 
 const (
-	nodeSelectorAnnotation = "open-cluster-management/nodeSelector"
-	tolerationsAnnotation  = "open-cluster-management/tolerations"
+	nodeSelectorAnnotation      = "open-cluster-management/nodeSelector"
+	tolerationsAnnotation       = "open-cluster-management/tolerations"
+	kubeVersionClusterClaimName = "kubeversion.open-cluster-management.io"
```
Comment (qiujian16) [inline pkg/helpers/helpers.go:69]: you can actually get kube verison directly from status of managed cluster, right?

Comment (elgnay) [reply pkg/helpers/helpers.go]: Yes, I've updated the code as suggested.

File: pkg/constants/constants.go [hunk 2]
```diff
@@ -136,3 +136,7 @@ const (
 	AutoImportSecretRosaConfigClusterIDKey  string            = "cluster_id"
 	AutoImportSecretRosaConfigRetryTimesKey string            = "retry_times"
 )
+
+const (
+	DefaultKlusterletPriorityClassName = "multicluster-engine-agent"
```
Comment (qiujian16) [inline pkg/constants/constants.go:141]: should we just call it klusterlet?

Comment (elgnay) [reply pkg/constants/constants.go]: Renamed.

**Pull request merged**